bitnami/chainloop/values.yaml

```diff
@@ -0,0 +1,2046 @@
+      ## Specifies whether recommended updates are applied when a Pod is started and whether recommended updates are applied during the life of a Pod
+      ## Possible values are "Off", "Initial", "Recreate", and "Auto".
+      ##
+      updatePolicy:
+        updateMode: Auto
+    ## @param cas.autoscaling.hpa.enabled Enable HPA for cas pods
+    ## @param cas.autoscaling.hpa.minReplicas Minimum number of replicas
+    ## @param cas.autoscaling.hpa.maxReplicas Maximum number of replicas
+    ## @param cas.autoscaling.hpa.targetCPU Target CPU utilization percentage
+    ## @param cas.autoscaling.hpa.targetMemory Target Memory utilization percentage
+    ##
+    hpa:
+      enabled: false
+      minReplicas: ""
+      maxReplicas: ""
+      targetCPU: ""
+      targetMemory: ""
+
+
+## @section Dependencies
+# ##################################
+# #          Dependencies          #
+##################################
+
+## PostgreSQL chart configuration
+## ref: https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml
+## @param postgresql.enabled Switch to enable or disable the PostgreSQL helm chart
+## @param postgresql.auth.enablePostgresUser Assign a password to the "postgres" admin user. Otherwise, remote access will be blocked for this user
+## @param postgresql.auth.username Name for a custom user to create
+## @param postgresql.auth.password Password for the custom user to create
+## @param postgresql.auth.database Name for a custom database to create
+## @param postgresql.auth.existingSecret Name of existing secret to use for PostgreSQL credentials
+postgresql:
+  enabled: true
+  auth:
+    enablePostgresUser: false
+    username: "chainloop"
+    password: "chainlooppwd"
+    database: "chainloop-cp"
+    existingSecret: ""
+
+# Vault server running in development mode --set development=true
+# IMPORTANT: This is not meant to run in production
+
+## Bitnami Hashicorp Vault chart configuration
+## ref: https://github.com/bitnami/charts/blob/main/bitnami/vault/values.yaml
+## @param vault.server.args Arguments to pass to the vault server. This is useful for setting the server in development mode
+## @param vault.server.config Configuration for the vault server. Small override of default Bitnami configuration
+## @param vault.server.extraEnvVars[0].name Root token for the vault server
+## @param vault.server.extraEnvVars[0].value The value of the root token. Default: notasecret
+## @param vault.server.extraEnvVars[1].name Address to listen on development mode
+## @param vault.server.extraEnvVars[1].value The address to listen on. Default: [::]:8200
+vault:
+  server:
+    args: [
+      "server",
+      "-dev"
+    ]
+    extraEnvVars:
+      - name: VAULT_DEV_ROOT_TOKEN_ID
+        value: "notasecret"
+      - name: VAULT_DEV_LISTEN_ADDRESS
+        value: "[::]:8200"
+    config: "storage \"inmem\" {}\ndisable_mlock = true\nui = true\nservice_registration \"kubernetes\" {}"
```

Comment:
I coudn't find any references to `vault.server.args` or `vault.server.config` in the templates. Are these values used anywhere?